src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -877,9 +877,7 @@ def new_chunk():
 
     if previous_tokens:
         if return_timestamps:
-            # Last token should always be timestamps, so there shouldn't be
-            # leftover
-            raise ValueError(
+            logger.warning(
                 "There was an error while processing timestamps, we haven't found a timestamp as last token. Was"
                 " WhisperTimeStampLogitsProcessor used?"
             )
```

Comment:
Since you suspect that the model was probably not finetuned with timestamps we can also add this

Reply:
I hesitated to put it here. While I have 1 example where I *think* it's true, I don't think it's enough to make it the first and foremost recommendation.

Reply:
(I occasionally hit this error when using the pre-trained model in Flax, so it's not specific to fine-tuned ones!)

Reply:
Interesting to know  !